packages/sdk-metrics/src/export/MetricReader.ts

```diff
@@ -139,9 +149,36 @@ export abstract class MetricReader {
       throw new Error('MetricReader is shutdown');
     }
 
-    return this._metricProducer.collect({
+    const collectionOptions = {
       timeoutMillis: options?.timeoutMillis,
-    });
+    };
+    const [sdkCollectionResults, ...additionalCollectionResults] =
+      await Promise.all([
+        this._sdkMetricProducer.collect(collectionOptions),
+        ...this._metricProducers.map(producer =>
+          producer.collect(collectionOptions)
```

Comment:
It would be problematic if the provided metric producers modified the input `collectionOptions`. I'd recommend constructing the option in place to avoid churn.